org.kde.kwave.json

```
@@ -0,0 +1,91 @@
+{
+    "id": "org.kde.kwave",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "6.8",
+    "sdk": "org.kde.Sdk",
+    "command": "kwave",
+    "finish-args": [
+        "--device=dri",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=pulseaudio",
+        "--socket=wayland",
+        "--system-talk-name=org.freedesktop.UDisks2"
+    ],
+    "cleanup": [
+        "/include",
+        "/lib/cmake",
+        "/lib/pkgconfig",
+        "/share/man",
+        "*.la",
+        "*.a"
+    ],
+    "modules": [
+        {
+            "name": "libaudiofile",
+            "buildsystem": "autotools",
+            "config-opts": ["--disable-docs", "--disable-examples"],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/mpruett/audiofile",
+                    "commit": "b62c902dd258125cac86cd2df21fc898035a43d3"
+                }
+            ],
+            "cleanup": ["/bin"]
+        },
+        {
+            "name": "id3lib",
+            "buildsystem": "autotools",
+            "config-opts": ["--disable-static"],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "http://downloads.sourceforge.net/id3lib/id3lib-3.8.3.tar.gz",
+                    "sha256": "2749cc3c0cd7280b299518b1ddf5a5bcfe2d1100614519b68702230e26c7d079"
+                },
+                {"type": "patch", "path": "patches/00-add-libz-to-compilation.patch"},
+                {"type": "patch", "path": "patches/05-create-m4-directory.patch"},
+                {"type": "patch", "path": "patches/10-fix-compilation-with-cpp-headers.patch"},
+                {"type": "patch", "path": "patches/15-fix-headers-of-main-functions.patch"},
+                {"type": "patch", "path": "patches/30-fix-utf16.patch"},
+                {"type": "patch", "path": "patches/31-fix-utf16-stringlists.patch"},
+                {"type": "patch", "path": "patches/40-deal-with-mkstemp.patch"},
+                {"type": "patch", "path": "patches/50-remove-outdated-check.patch"},
+                {"type": "patch", "path": "patches/60-add-c-wrapper-functions.patch"},
+                {"type": "patch", "path": "patches/60-fix_make_check.patch"},
+                {"type": "patch", "path": "patches/60-id3lib-missing-nullpointer-check.patch"},
+                {"type": "patch", "path": "patches/61-fix_vbr_stack_smash.patch"},
+                {"type": "patch", "path": "patches/62-remove-remote-icons.patch"}
```

```suggestion
                {"type": "patch", "path": "patches/62-remove-remote-icons.patch"}
                {"type": "patch", "path": "patches/id3lib-3.8.3-autoreconf.patch"},
                {"type": "shell", "commands": ["autoreconf -fi"]}
```

The patch is from here: https://github.com/flathub/org.gnome.EasyTAG/blob/master/id3lib-3.8.3-autoreconf.patch